src/Console/src/Usage/Input.php

```diff
@@ -60,10 +62,10 @@ public function option($option)
         return $this->getOption($option);
     }
 
-    public function getOption($option)
+    public function getOption($option): OptionValue
     {
         if (!isset($this->options[$option])) {
-            return null;
```

Comment:
Throw exception rather than return `null` requesting an unknown option. 
In theory the `OptionDefinitionParser` should provide default value for each option, so this should never happen.